modules/openapi-generator/src/main/resources/swift5/Podspec.mustache

```diff
@@ -33,14 +26,14 @@ Pod::Spec.new do |s|
   s.documentation_url = '{{.}}'
   {{/podDocumentationURL}}
   s.source_files = '{{swiftPackagePath}}{{^swiftPackagePath}}{{#useSPMFileStructure}}Sources/{{projectName}}{{/useSPMFileStructure}}{{^useSPMFileStructure}}{{projectName}}/Classes{{/useSPMFileStructure}}{{/swiftPackagePath}}/**/*.swift'
-  s.dependency 'AnyCodable-FlightSchool', '~> 0.6.1'
+  s.dependency 'AnyCodable-FlightSchool', '~> 0.6.6'
   {{#useAlamofire}}
-  s.dependency 'Alamofire', '~> 5.4.3'
+  s.dependency 'Alamofire', '~> 5.6.2'
```

Comment:
I wonder whether we could be a bit less restrictive here and only pin major versions instead of major and minor version. With `~> 5.6.2` we only allow 5.6.2 and all following patch updates for it. This might not be a problem if there is no other dependency to Alamofire in a project but otherwise a project could never resolve Alamofire 5.7+ versions and is forced to stay on minor version 5.6* until generator is updated again. As Alamofire version policy is also based on (semantic versioning)[https://github.com/Alamofire/Alamofire/blob/master/CHANGELOG.md] this might be ok I think. Just a thought that crossed my mind

Reply:
Hey. I would probably still allow the lower AF versions as I know some dependencies still require 5.4*. That's why my suggestion would be to use `~> 5.4` modifier to not break dependency resolution for those.

Reply:
Done 👍

Reply:
`~> 5.4.0` will only allow up to the next minor version, so e.g. AF 5.5.0 would already not be possible to resolve anymore. I think in order to support everything from 5.4.0 until (and not including) the next major version we would need to use `~> 5.4` instead. Then we would also allow 5.5.*, 5.6.* and so on

Reply:
CI is failing... Maybe I will try to revert the dependencies changes and save that for another PR

Reply:
Yes agree its probably better to do this in a separate PR

Reply:
@Jonas1893 thanks for helping review this PR. If you want to help review more Swift Cliente PRs are you welcome 👍

Reply:
Hey @4brunu thx will gladly do so :)